config/metadata/configMeta.yaml

```diff
@@ -935,7 +935,8 @@ groups:
           allocation and `MaxAlloc` will be used instead. If set, then this must
           be a memory size. 64-bit values are supported. Sizes with standard
           unit suffixes (`MB`, `GiB`, etc.) and Kubernetes units (`M`, `Gi`, 
-          etc.) are also supported.
+          etc.) are also supported. If set, `Collections.maxAlloc` must not be
```

Comment:
```suggestion
          etc.) are also supported. If set, `Collections.MaxAlloc` must not be
```

Reply:
Ops thanks, I also found that the `AcceptOnlyListedKeys` docs need to be changed as well as it currently has the same issue (using requiredWith along with a default).